lib/error.js

```diff
@@ -4,14 +4,12 @@ var util = require('util'),
     Controller = require('hmpo-form-controller');
 
 function getArgs(type, args) {
-    if (type === 'maxlength') {
-        return { maxlength: args[0] };
-    } else if (type === 'minlength') {
-        return { minlength: args[0] };
-    } else if (type === 'exactlength') {
-        return { exactlength: args[0] };
-    } else if (type === 'past') {
+    if (type === 'past') {
         return { age: args.join(' ') };
+    } else if (args !== undefined) {
```

Comment:
Should there be some validation of 'type' going on here?
Presumably we should check that it is a string with size > 0?

Might be nice to validate args too but that should probably be higher up in the code. (And to be fair the original code didn't bother. :-/ )

Perhaps we should instead have a block of code at the top of the function that throws an exception when called with invalid arguments. (That's a bug in the calling code of course.) Rather than failing silently which is what we seem to be doing currently.

@gavboulton, @easternbloc, @JoeChapman do any of you have any strong opinions on this?